tracker.rst

```diff
@@ -136,6 +136,22 @@ In the sidebar you can also find links to summaries for easy issues and
 issues with a patch.
 
 
+Disagreement With a Resolution on the Issue Tracker
+===================================================
+
+First, take some time to consider any comments made in association with the
+resolution of the tracker issue. On reflection, they may seem more reasonable
+than they first appeared.
+
+If you still feel the resolution is incorrect, then raise the question on
+`python-dev`_. Further argument on `python-dev`_ after a consensus has been
+reached amongst the core developers is unlikely to win any converts.
+
+You should not reopen issues that have been closed by a core developer.
```

Comment:
@mariatta This is fine as is. A slight refinement might be more direct and welcoming: `Issues closed by a core developer have already been carefully considered. Please do not reopen a closed issue.`